src/main/java/uk/gov/hmcts/reform/professionalapi/configuration/MethodValidationPostProcessorConfig.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.professionalapi.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.validation.beanvalidation.MethodValidationPostProcessor;
+
+@Configuration
+public class MethodValidationPostProcessorConfig {
+
+    @Bean
+    public MethodValidationPostProcessor methodValidationPostProcessor() {
+        return new MethodValidationPostProcessor();
+    }
+}
```

Comment:
_MethodValidationPostProcessor:_

> A convenient BeanPostProcessor implementation that delegates to a JSR-303 provider for performing method-level validation on annotated methods.
> 
> Target classes with such annotated methods need to be annotated with Spring’s Validated annotation at the type level, for their methods to be searched for inline constraint annotations. Validation groups can be specified through @Validated as well. By default, JSR-303 will validate against its default group only.

Reply:
This is required if we wish to use @Pattern on @PathVariable in the Controller rather than calling a validator method.